### PR TITLE
Don't import translation strings with mismatched tokens, fix Explore & Places tab bar localized titles

### PR DESCRIPTION
**Pull request by joewalsh**

### CODE REVIEW

File: WikipediaUnitTests/Code/TWNStringsTests.m [hunk 1]
```diff
@@ -225,6 +226,29 @@ - (void)testiOSTranslationStringForBracketSubstitutions {
                         XCTAssertTrue(false, @"Unsupported {{ }} in localization");
```
Comment (montehurd) [inline WikipediaUnitTests/Code/TWNStringsTests.m]: Could the assert string report which localization failed?

File: WikipediaUnitTests/Code/TWNStringsTests.m [hunk 2]
```diff
@@ -209,7 +208,9 @@ - (void)testIncomingTranslationStringForBracketSubstitutions {
     }
 }
 
-- (void)testiOSTranslationStringForBracketSubstitutions {
+- (void)testiOSTranslationStringForBracketSubstitutionsAndMismatchedTokens {
```
Comment (montehurd) [inline WikipediaUnitTests/Code/TWNStringsTests.m]: Thoughts on maybe keeping this as two separate tests and possibly a utility method for any common bits?

Comment (joewalsh) [inline WikipediaUnitTests/Code/TWNStringsTests.m]: The failures are logged separately, so that's enough for me.

Comment (montehurd) [inline WikipediaUnitTests/Code/TWNStringsTests.m]: Ya, was thinking more in terms of keeping tests "unit"y if it wasn't a hassle.

**Pull request merged**